java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -4193,6 +4193,206 @@ void testContainsReEmptyInput() {
+         ColumnVector expected2 = ColumnVector.fromBoxedBooleans(
+           true, true, true, true, true, true, true, true, true, true);
+         ColumnVector expected3 = ColumnVector.fromBoxedBooleans(
+           false, true, false, false, false, false, false, false, false, false);
+         ColumnVector expected4 = ColumnVector.fromBoxedBooleans(
+           false, false, true, false, false, false, false, false, false, false);
+         ColumnVector expected5 = ColumnVector.fromBoxedBooleans(
+           false, false, false, false, false, false, true, false, false, false);
+         ColumnVector expected6 = ColumnVector.fromBoxedBooleans(
+           false, false, true, true, false, false, true, false, false, false);
+         ColumnVector expected7 = ColumnVector.fromBoxedBooleans(
+           true, true, true, true, false, false, false, false, false, true);
+         ColumnVector expected8 = ColumnVector.fromBoxedBooleans(
+           false, false, false, false, false, false, false, false, true, false);
+         ColumnVector expected9 = ColumnVector.fromBoxedBooleans(
+           false, false, true, true, false, false, true, false, false, false);
+         ColumnVector expected10 = ColumnVector.fromBoxedBooleans(
+           true, true, true, true, false, false, true, false, false, false);
+         ColumnVector expected11 = ColumnVector.fromBoxedBooleans(
+           true, true, true, true, false, false, true, false, false, true);
+         ColumnVector expected12 = ColumnVector.fromBoxedBooleans(
+           false, true, true, true, false, false, true, false, false, false);
+         ColumnVector expected13 = ColumnVector.fromBoxedBooleans(
+           false, false, false, true, false, false, false, false, false, false);
+         ColumnVector expected14 = ColumnVector.fromBoxedBooleans(
+           false, false, false, true, false, false, false, false, false, true);
+         ColumnVector expected15 = ColumnVector.fromBoxedBooleans(
+           false, false, false, false, false, false, false, false, false, false);
+         ColumnVector expected16 = ColumnVector.fromBoxedBooleans(
+           false, false, false, false, false, false, false, false, false, true)) {
+      assertColumnsAreEqual(expected1, res1);
+      assertColumnsAreEqual(expected2, res2);
+      assertColumnsAreEqual(expected3, res3);
+      assertColumnsAreEqual(expected4, res4);
+      assertColumnsAreEqual(expected5, res5);
+      assertColumnsAreEqual(expected6, res6);
+      assertColumnsAreEqual(expected7, res7);
+      assertColumnsAreEqual(expected8, res8);
+      assertColumnsAreEqual(expected9, res9);
+      assertColumnsAreEqual(expected10, res10);
+      assertColumnsAreEqual(expected11, res11);
+      assertColumnsAreEqual(expected12, res12);
+      assertColumnsAreEqual(expected13, res13);
+      assertColumnsAreEqual(expected14, res14);
+      assertColumnsAreEqual(expected15, res15);
+      assertColumnsAreEqual(expected16, res16);
```

Comment:
Similar to above.

In addition, the Java tests are just to verify that you can successfully call the native cudf APIs. So just a couple of tests (2, 3) would be enough. Too much tests are redundant because cudf already has extensive unit tests.

Reply:
Updated. Thank you!